src/cli.ts

```diff
@@ -131,15 +132,18 @@ async function performFirstPackAndCopyToCallForGivenArguments(
 
   await executePackageTargetAndCopyToDestinationDirectoriesCalls(excludedDestinationPaths, logger, packAndCopyCalls);
 
+  const yarnrcYmlExistence: Promise<boolean> = doesYarnrcYmlFileExist();
   await maybeOutputNextStepsText(
     maybeDestinationDirectoryToAddDependencyOn,
     rootTargets,
     outputPostCommandMessages,
     logger,
     process.cwd(),
-    doesPackageLockFileExist,
-    doesYarnLockFileExist,
-    () => tryGetProjectPackageJsonContent(path.resolve('./'))
+    doesPackageLockFileExist(),
+    doesYarnLockFileExist(),
+    yarnrcYmlExistence,
+    () => tryGetProjectPackageJsonContent(path.resolve('./')),
+    () => isYarnBerryUsingNodeModulesLinker(yarnrcYmlExistence)
```

Comment:
However, I continued to delay file parsing since it's relatively heavier and not useful in all conditional branches.